### DISCUSSION TITLE
Cobbler forward client to WDS server?

### DISCUSSION THREAD

**Discussion by easpeagle** [cobbler/cobbler#3553] [Q&A] [answered]
Hey Folks,

Is there a way to configure the dhcp.template or grub.template to forward a client system to a WDS server for OS deployment outside of Cobbler. I'm guessing that I'll have to use iPXE for UEFI systems. Just need a clue about what this would look like in Cobbler.

Thanks.

**Comment by SchoolGuy** [comment, marked as answer]
@easpeagle Since in the end the WDS server has a TFTP server you just need to point your clients that use the WDS server to a different next server. This can be done on a per-profile basis or a per-system basis. If your DHCP server is managed externally of Cobbler you need to define either a group, subnet or a list of client IDs that are being redirected to the TFTP server from the WDS deployment.

However I think @tpw56j is our expert in all things Windows. :)

**Comment by easpeagle** [comment]
@SchoolGuy ... yeah, I was landing on that as an option. Just creating a profile that overrides DHCP options 66 & 67 should cut it. Thanks for the clarity.
